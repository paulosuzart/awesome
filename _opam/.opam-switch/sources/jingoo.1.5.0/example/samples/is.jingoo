- 6 is divisibleby 4: {{ 6 is divisibleby 4 }}
- 6 is divisibleby (4): {{ 6 is divisibleby (4) }}
- divisibleby(3,6): {{ divisibleby(3,6) }}
- 3 is iterable: {{ 3 is iterable }}
- [1,2,3] is iterable: {{ [1,2,3] is iterable }}
- "hoge" is iterable: {{ "hoge" is iterable }}
- null is iterable: {{ null is iterable }}
