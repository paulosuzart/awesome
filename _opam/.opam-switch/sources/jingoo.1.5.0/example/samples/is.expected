- 6 is divisibleby 4: false
- 6 is divisibleby (4): false
- divisibleby(3,6): true
- 3 is iterable: false
- [1,2,3] is iterable: true
- "hoge" is iterable: true
- null is iterable: true
